prime-router/docs/openapi.yml

```diff
@@ -9,6 +9,160 @@ info:
   version: 0.2.0-oas3
 paths:
 
+  # The sender end-point handles receive actions.
+  #
+  # END-POINTS
+  # /submissions
+  # /submissions/submission/{reportid}
+  #
+  # ALT END-POINT OPTIONS
```

Comment:
Not sure what ALT end-points mean this case.

Reply:
@RickHawesUSDS these were alternate options being captured that came out of Slack discussions. Ticket #2272 will resolve these.